variables.js

```diff
@@ -0,0 +1,78 @@
+// 1. Age Calculator:
+//     * Store your birth year in a variable.
+//     * Store a future year in a variable.
+//     * Calculate your 2 possible ages for that year based on the stored values.
+//     * For example, if you were born in 1988, then in 2026 you'll be either 37 or 38, depending on what month it is in 2026.
+//     * Log them to the screen like so: "I will be either `ageMin` or `ageMax` in `futureYear`", substituting the values.
+
+const birthYear = 1987;
+let futureYear = 2028;
+ageMax = futureYear - birthYear;
+ageMin = ageMax--;
+console.log('I will be either ' + ageMin + ' or ' + ageMax + ' in ' + futureYear+ '.');
+
+// 2. Snack Supply Calculator:
+//     * Store your current age in a variable.
+//     * Store a maximum age in a variable.
+//     * Store an estimated snack amount per day (as a number).
+//     * Calculate how many snacks you would eat total, from your current age until the maximum age.
+//     * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
+
+let currentAge = 31;
+let maxAge = 91;
+let snackDaily = 2;
+let snackTotal = (maxAge - currentAge) * 2;
```

Comment:
Since it's asking for snacks you need everyday, maybe you need to multiply the whole number by 365.